### PR TITLE
Merge fetch of VL and NV by changing data type

### PR DESCRIPTION
**Pull request by flomillot**
Change fetchVoltageLevelsListInfos to fetchVoltageLevelsMapInfos for better data handling and efficiency. Update equipment-filter.jsx to implement new function and handle nominal voltages accordingly.

### CODE REVIEW

**Review by thangqp — APPROVED** [review]
Code OK, Test OK

File: src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx [hunk 1]
```diff
@@ -96,11 +93,16 @@ const EquipmentFilter = forwardRef(
...
                 .then((voltageLevels) => {
                     const vlMap = new Map();
-                    voltageLevels.forEach((vl) => vlMap.set(vl.id, vl.name));
+                    const nvSet = new Set();
+                    voltageLevels.forEach((vl) => {
+                        vlMap.set(vl.id, vl.name);
+                        nvSet.add(vl.nominalV);
+                    });
                     setVoltageLevelsMap(vlMap);
+                    setNominalVoltages([...nvSet.values()].sort());
```
Comment (thangqp) [inline src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx:105]: sort is not effective for number, it seems sort for string

File: src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx [hunk 2]
```diff
@@ -96,11 +93,16 @@ const EquipmentFilter = forwardRef(
...
         useEffect(() => {
             // Load voltage level IDs
-            fetchVoltageLevelsListInfos(studyUuid, currentNode.id)
+            fetchVoltageLevelsMapInfos(studyUuid, currentNode.id)
                 .then((voltageLevels) => {
                     const vlMap = new Map();
-                    voltageLevels.forEach((vl) => vlMap.set(vl.id, vl.name));
+                    const nvSet = new Set();
+                    voltageLevels.forEach((vl) => {
+                        vlMap.set(vl.id, vl.name);
```
Comment (SlimaneAmar) [inline src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx:101]: Why 2 collections ?
Just one map : id -> vl ?

Comment (thangqp) [reply src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx:101]: must aggregate nominal V to show in the separated criteria..

File: src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx (hunk 1 above)
Comment (flomillot) [reply src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx:105]: Works on Chromium bases browsers 
image

Comment (flomillot) [reply src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx:105]: My bad, I change it

File: src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx (hunk 2 above)
Comment (flomillot) [reply src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx:101]: The Set contains only unique elements

**Review by thangqp — APPROVED** [review]
Code OK, Test OK

**Pull request merged**